### PR TITLE
Enable cron 4.15 hcp and also reverting HCP tests back to us-west-2 region

### PR DESCRIPTION
**Pull request by svetsa-rh**
Previously we didn't enable 4.15 tests as the version was not available for HCP cluster creation - see https://github.com/openshift/release/pull/44179#issuecomment-1764911083 

Verified OCP 4.15 is enabled for Rosa - See https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/45058/rehearse-45058-pull-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.15-nightly-x86-perfscale-rosa-hcp-1zone-data-path-2nodes-periodic/1719151368830193664/artifacts/perfscale-rosa-hcp-1zone-data-path-2nodes-periodic/rosa-cluster-provision/build-log.txt 

Verified us-west-2 is working after new deployments few hours ago https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.14-nightly-x86-perfscale-rosa-hcp-3zones-node-density-heavy-24nodes-periodic/1719392294437654528 

Justification for Cron - Given the breadth of tests to run, decided to run 4.15 HCP small scale tests on Friday evening as the results analysis can wait until Monday for these tests.

Jira for reference: https://issues.redhat.com/browse/PERFSCALE-2607

### CODE REVIEW

**Comment by svetsa-rh** [comment]
Running make update rn

**Comment by paigerube14** [comment]
Can we run a hcp test without the us-east-1 region specified and the 2 node data path test to verify these work properly

**Comment by svetsa-rh** [comment]
@krishvoor @paigerube14 As suggested by Paige, ran the 2 node data path hcp test without the us-east-1 region specified.
The cluster was provisioned successfully - https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/45117/rehearse-45117-periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-rosa_hcp-4.15-nightly-x86-perfscale-rosa-hcp-1zone-data-path-2nodes-periodic/1719430177773064192/artifacts/perfscale-rosa-hcp-1zone-data-path-2nodes-periodic/rosa-cluster-provision/build-log.txt

Issue we are seeing is in network test, which needs to be handled separately (unrelated to this PR).

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-nightly-x86.yaml [hunk 1]
```diff
@@ -40,8 +40,8 @@ tests:
     - chain: openshift-qe-control-plane
     workflow: rosa-aws-sts-hypershift
   timeout: 5h0m0s
-- always_run: false
-  as: perfscale-rosa-hcp-3zones-node-density-heavy-24nodes-periodic
+- as: perfscale-rosa-hcp-3zones-node-density-heavy-24nodes-periodi
```
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-nightly-x86.yaml:43]: periodic

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-nightly-x86.yaml [hunk 2]
```diff
@@ -40,8 +40,8 @@ tests:
...
+  cron: 0 20 * * 5
```
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-nightly-x86.yaml:44]: Just once a week?

**Review by jtaleric — COMMENTED** [review]
Couple of nits -- Do we not just want to call out the region we are testing in?

**Comment by svetsa-rh** [comment]
> Couple of nits -- Do we not just want to call out the region we are testing in?

@jtaleric No, not necessary. We have default region for perfscale profiles set to `us-west-2`. So, unless there is specific purpose to run outside of this, no need to set.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-nightly-x86.yaml (hunk 1 above)
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-nightly-x86.yaml]: Corrected.

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-nightly-x86.yaml (hunk 2 above)
Comment (svetsa-rh) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-nightly-x86.yaml:44]: yes, for self managed we can run more often if needed, but for managed once a week should suffice. I confirmed with Roshni. 
If we ever find once a week is not enough, we can bump it up.

**Review by jtaleric — APPROVED** [review]
/lgtm

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-nightly-x86.yaml (hunk 2 above)
Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa_hcp-4.15-nightly-x86.yaml:44]: We can run these rosa-hcp jobs once weekly this should be good to start with.

**Pull request merged**